src/views/mfa-verify/PushForm.js

```diff
@@ -49,6 +63,19 @@ define(['okta'], function (Okta) {
         factorName: this.model.get('factorLabel'),
         deviceName: this.model.get('deviceName')
       });
+
+      var that = this;
+      if (that.enabled) {
+        window.setTimeout(function () {
```

Comment:
turns out `submit` method could not take effect during postRender phase, staying with `setTimeout` for dom manipulation.